HIP/hip-583.md

```diff
@@ -10,220 +10,451 @@ status: Accepted
+#### Proof of Ownership
 
-Since Hedera accounts require keys to ensure security, a matching `ECDSA` `public-key` must be provided in a future signed transaction issued by the owner of the `public-address`. Due to the nature of `ECDSA` keys the `public-key` and `public-address` may be extracted from the signature of a transaction signed by the `ECDSA` private key that owns the new account. This process will complete the creation of the account, taking it from a `Hollow Account` to a normal complete account with a valid `public-key`.
-Note, in this case the final value of the account alias will remain as the `public-address` and not the `public-key` as it would be in the `auto-create` case.
+If the `alias` is used during `CryptoTransfer` as the _target_ of the transfer, then there is no need to prove that
+anyone controls that account. If an `alias` is used in the `AccountID` of the payer, then ownership must be asserted,
+just as is done for any other transaction today.
 
-As an illustration, a wallet or exchange could create and fund a new Hedera account based on the `public-address` alone. Once the transaction goes through the owner of the `public-address` user may use a wallet of their choice to submit a signed transaction (e.g. `CryptoTransfer`, `ContractCreate`, `ContractCall`, `EthereumTransaction` or `TokenAssociation`) to claim ownership of the account and carry out the desired transaction. The ownership claim and account completion will be seamless and transparent to the user.
+#### Validation of the `alias`
 
-`CryptoTransfer` transactions should also support this case.
-This will support users, developers and exchanges who prefer to create and credit an accounts balance in one step.
+It **is** possible to specify an `account-num alias` as the _target_ of a `CryptoTransfer` transaction. Indeed, for
+Ethereum-native wallets, this is the only way to transfer value to Hedera accounts that don't have an
+`evm address alias`. However, it is **impossible** to auto-create an account using the `account-num alias`. If the
+account does not exist, the transfer will fail.
 
-`EthereumTransaction` transactions where value is transfered  (i.e `tx.to` and `tx.value` are set) should also support this case.
-This will support users of `Ethereum-native wallets` and eth dev tools who are accumstomed to creating and crediting an accounts balance in one step.
-In this case th `tx.to` value of the RLP encoded transaction would dictate the `public-address` to be used as an alias.
+##### `key alias`
```

Comment:
It seems like this section was copied from the `CryptoCreate` section above. I think we don't need `key alias` section for `CryptoTransfer` because those cases are described in HIP-32 and we are not making any changes in HIP-583 for `CryptoTransfer` with key aliases.

Reply:
It is not an exactly copy-paste, there are important differences in the behavior. While HIP-32 does define this capability. I wanted to be explicit with how this works with `evm address alias`, which is new in this HIP. You will notice, for example:

> 4. If the `Key` is an `ECDSA_secp256k1` key:
>     1. If the alias is already in use, we use that account and do not create a new one
>     2. If the derived `evm address` from the key is already in use, we use that account and do not create a new one
>     3. If the alias is not in use, we create a new account and use the given key as the key on the account.

The second bullet explicitly calls out the behavior with evm address aliases involved. I any event, I want to be explicit here to help define the test cases that must be written and verified before the feature is complete.